Challenge/julio_bertty_sierra/script.js

```diff
@@ -0,0 +1,7 @@
+let frutas = ['Pera', 'Manzana', 'Sandia', 'Banana', 'Mora', 'Lulo', 'Tamarindo', 'Uva'];
+let index = 0;
+console.log('** Vector de Frutas **')
+while (index < frutas.length) {
+    console.log("Fruta "+ (index + 1) +": "+ frutas[index]);    
+    index+=1
```

Comment:
Acá falta el **`;`** al final de la instrucción. Es posible que no te haya marcado error al ejecutarlo ya que el uso de ";" en javascript es opcional, yo a manera personal te sugiero usar ";". En el mundo laboral hay que adaptarse ya que puede ser que sea obligatorio que lo uses o no, el líder técnico del proyecto te lo notificará o en su defecto preguntar si lo puedes usar o no en el proyecto. Para este curso estaremos utilizando `**;**`.
Acá te dejo un link de las cosas a tomar en cuenta en caso de que tu preferencia sea no utilizar el ";"
https://dev.to/adriennemiller/semicolons-in-javascript-to-use-or-not-to-use-2nli